feeds/wlan-ap/opensync/files/bin/wlan_ap_rrm_rebalance.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+filename="/tmp/wireless_changes.txt"
+freq_table="1:2412 2:2417 3:2422 4:2427 5:2432 6:2437 7:2442 8:2447 9:2452 10:2457 11:2462 12:2467 13:2472 14:2484 36:5180 40:5200 44:5220 48:5240 52:5260 56:5280 60:5300 64:5320 100:5500 104:5520 108:5540 112:5560 116:5580 120:5600 124:5620 128:5640 132:5660 136:5680 140:5700 144:5720 149:5745 153:5765 157:5785 161:5805 165:5825"
+while IFS= read -r line; do
+        if [ "$line" = "phy0" ]; then
+                echo $line
+                radio="radio0"
+                interface="wlan0"
+        elif [ "$line" = "phy1" ]; then
+                echo $line
+                radio="radio1"
+                interface="wlan1"
+        elif [ "$line" = "phy2" ]; then
+                echo $line
+                radio="radio2"
+                interface="wlan2"
+        fi
+done< "$filename"
+
+channel=$(uci get wireless.${radio}.channel)
+echo $channel
+for freq in $freq_table ; do
+        chan="${freq%%:*}"
+        frequency="${freq##*:}"
+        jstr='{"freq":'"$frequency"'}'
+        if [ "${channel}" = "${chan}" ]; then
+                ubus call hostapd.${interface} switch_chan "$jstr"
```

Comment:
In rrm_ubus.c/ubus_set_channel_switch(), Nagendrababu found it necessary to include additional parameters to get this to work (especially for WIFI6).  In particular, I don't see the "bcn_count", the "bandwidth" or the "sec_channel_offset". 

Did you test on a WIFI6 as well as WIFI5?
Did you actual sniff the air for the CSA in the beacon?